index.js

```diff
@@ -16,39 +17,46 @@ app.get('/', (req, res) => {
 })
 
 // Despegar
-app.get('/city/:city', (req, res) => {
-  const cityParam = req.params.city
-  if (!cityParam) return res.json(null)
-  if (cityParam === 'Santiago' || cityParam === 'santiago') {
-    res.json({ error: 'No hay vuelos de Santiago a Santiago :retard:' })
-    return
-  }
-  if (cityParam === 'Rancagua' || cityParam === 'rancagua') {
-    res.json({ error: 'Rancagua no existe :retard:' })
-    return
+app.get('/city/:destination/:origin?', (req, res) => {
+  const { origin = 'santiago', destination } = req.params
+
+  if (!destination) return res.json(null)
+
+  if (destination === origin) {
+    res.json({ error: `No hay vuelos de ${origin} a ${destination} :retard:` })
   }
 
   ;(async () => {
-    let foundCities = []
     try {
-      foundCities = await utils.findCities(cityParam)
-      const cityExist = typeof foundCities !== 'undefined' && foundCities.length > 0
-      if (!cityExist) {
-        res.json({ error: 'Aún no tengo vuelos a esa ciudad' })
+      const destinationCities = await utils.findCities(destination)
+      const originCities = origin === 'santiago'
+        ? [{ code: 'scl' }]
+        : await utils.findCities(origin)
+
+      const originExist = typeof originCities !== 'undefined' && originCities.length > 0
+      const destinationExist = typeof destinationCities !== 'undefined' && destinationCities.length > 0
+
+      if (!originExist) {
+        res.json({ error: `Aún no tengo vuelos desde ${origin}` })
+      } else if (!destinationExist) {
+        res.json({ error: `Aún no tengo vuelos hasta ${destination}` })
       } else {
         // Select the first city found
-        const cityCode = foundCities[0].code.toLocaleLowerCase()
-        const despegarUrl = `https://www.despegar.cl/vuelos/scl/${cityCode}/`
+        const originCityCode = originCities[0].code.toLocaleLowerCase()
+        const destinationCityCode = destinationCities[0].code.toLocaleLowerCase()
+        const despegarUrl = `https://www.despegar.cl/vuelos/${originCityCode}/${destinationCityCode}/`
         const browser = await puppeteer.launch({ args: ['--no-sandbox', '--disable-setuid-sandbox'] })
         const page = await browser.newPage()
         page.setViewport({ width: 1280, height: 1000 })
         await page.goto(despegarUrl, { waitUntil: 'domcontentloaded' })
         try {
-          const price = await page.evaluate(() => document.querySelector('#alerts .price-amount').textContent)
-          if (!price) {
+          const priceCLP = await page.evaluate(() => document.querySelector('#alerts .price-amount').textContent)
+          if (!priceCLP) {
             res.json({ error: 'No encuentro vuelos para esta ciudad' })
           }
-          res.json({ price: price.replace('$ ', ''), url: despegarUrl })
+          const { data: dolarToday } = await axios.get('https://mindicador.cl/api/dolar')
```

Comment:
I'm not so convinced about depending on another API to give a response. If `minindicador` fails we won't show the response that we got from despegar. 

I think we have to ways here. Adding an optional query parameter to enable the USD conversion or making sure that if you get an exception from minindicator this won't break the response.

Also, another option is making the request to despegar in panama so you will be able to see prices in USD dollars and you can buy them through that site too `https://www.despegar.com.pa/vuelos/scl/bog/`